### ISSUE TITLE
Can I add documentation on 

### ISSUE THREAD

**Issue by datatalking** [pytorch/pytorch#38354]

**Comment by ezyang** [comment]
We probably just have to set `long_description` in setup.py. Would be happy to accept a patch on this.

**Comment by ezyang** [comment]
cc @soumith @seemethere

**Comment by seemethere** [comment]
Yup setting a `long_description` should just work the next time we upload a package to `pypi`.

I wish we could just update it post-upload but that functionality was removed when they moved to the new PyPA warehouse. https://github.com/pypa/warehouse/issues/2170#issuecomment-361176123
